tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "moduleResolution": "node",
     "forceConsistentCasingInFileNames": true,
     "resolveJsonModule": true,
+    "sourceMap": true,
```

Comment:
whats this enabling?

Reply:
It's usually required to use the debugger.  Pretty mixed results anyway though. I'll remove.